From 591964ce83c12b7faf0aa65908106e767e35f524 Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Mon, 5 Oct 2015 11:41:03 +0200
Subject: [PATCH 097/117] actions: fix documented limit for btrfs labels

Our documentation of set-label says that labels for btrfs are limited
to 256 characters, while btrfs-filesystem(1) says:

  Note
  the maximum allowable length shall be less than 256 chars

which is indeed true, as trying to set a label with 256 characters
fails. Hence, fix our documentation to say that the limit is actually
255 characters.

Related to RHBZ#1164708.
---
 generator/actions.ml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/generator/actions.ml b/generator/actions.ml
index cc18036..f0c157d 100644
--- a/generator/actions.ml
+++ b/generator/actions.ml
@@ -9192,7 +9192,7 @@ be mounted when trying to set the label.
 
 =item btrfs
 
-The label is limited to 256 bytes and some characters are
+The label is limited to 255 bytes and some characters are
 not allowed.  Setting the label on a btrfs subvolume will set the
 label on its parent filesystem.  The filesystem must not be mounted
 when trying to set the label.
-- 
2.5.0

